prac_05/state_names.py

```diff
@@ -1,21 +1,18 @@
 """
 CP1404/CP5632 Practical
 State names in a dictionary
-File needs reformatting
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
 STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
-# print(STATE_NAMES)
 
-state = input("Enter short state: ").upper()
-while state != "":
-    if state in STATE_NAMES:
-        print(state, "is", STATE_NAMES[state])
+full_state = input("Enter short state: ").upper()
+while full_state != "":
+    if full_state in STATE_NAMES:
+        print(full_state, "is", STATE_NAMES[full_state])
     else:
         print("Invalid short state")
-    state = input("Enter short state: ").upper()
+    full_state = input("Enter short state: ").upper()
 
-for short_name ,state in STATE_NAMES.items():
-    print("{:<3} is {:<10}".format(short_name ,state))
\ No newline at end of file
+for short_state, full_state in STATE_NAMES.items():
```

Comment:
You've got the short/long correct here so only update the variable names (not the order you've got them listed)